2_process.R

```diff
@@ -164,7 +171,22 @@ p2_targets_list <- list(
                     setNames(gsub('_\\d{4}', '', names(.)))) %>%
                # rbind the list of cleaned dfs
       do.call(rbind, .)
+  ),
+  
+  # Process NHDv2 attributes referenced to cumulative upstream area;
+  # returns object target of class "list". List elements for CAT_PPT
```

Comment:
To further see what I mean here, run p2_nhdv2_attr_upstream[[28]], p2_nhdv2_attr_upstream[[29]], and p2_nhdv2_attr_upstream[[30]] once the pipeline is finished building.

p1_vars_of_interest_downloaded_csvs[30] only contains the "CAT_PPT" data because of the way these data were divvied up on ScienceBase. So because !"CAT" %in% cols only the PRMS_segid's will get returned. This is kinda ugly within this target but in the combined target they'll be functionally omitted.

Reply:
> run p2_nhdv2_attr_upstream[[28]], p2_nhdv2_attr_upstream[[29]], and p2_nhdv2_attr_upstream[[30]]

28 and 30 look the same - they only have PRMS_segid as a column. 29 also has the TOT_PPT columns for each month, but no CAT columns. Is that what you were expecting?

Reply:
Yep, that is what I was expecting. I tried to explain in my comment above - 28 and 30 are essentially empty because those downloaded data files do not have any columns that contain the prefix "TOT".